docs/architecture/adr-070-unordered-account.md

```diff
@@ -2,214 +2,137 @@
 
 ## Changelog
 
-* Dec 4, 2023: Initial Draft (@yihuang, @tac0turtle, @alexanderbez)
-* Jan 30, 2024: Include section on deterministic transaction encoding
+- Dec 4, 2023: Initial Draft (@yihuang, @tac0turtle, @alexanderbez)
+- Jan 30, 2024: Include section on deterministic transaction encoding
+- Mar 12, 2025: Revise implementation to use Cosmos SDK KV Store and require unique timeouts per-address (@technicallyty)
 
 ## Status
 
-ACCEPTED
+Draft
 
 ## Abstract
 
 We propose a way to do replay-attack protection without enforcing the order of
-transactions, without requiring the use of nonces. In this way, we can support
-un-ordered transaction inclusion.
+transactions and without requiring the use of monotonically increasing sequences. Instead, we propose
+the use of a time-based, ephemeral sequence.
 
 ## Context
 
-As of today, the nonce value (account sequence number) prevents replay-attack and
-ensures the transactions from the same sender are included into blocks and executed
-in sequential order. However it makes it tricky to send many transactions from the
-same sender concurrently in a reliable way. IBC relayer and crypto exchanges are
-typical examples of such use cases.
+Account sequence values serve to prevent replay-attacks and ensure transactions from the same sender are included into blocks and executed
+in sequential order. Unfortunately, this makes it difficult to reliably send many concurrent transactions from the
+same sender. Victims of such limitations include IBC relayers and crypto exchanges.
 
 ## Decision
 
-We propose to add a boolean field `unordered` to transaction body to mark "un-ordered"
-transactions.
+We propose adding a boolean field `unordered` and a uint64 field `timeout_timestamp` to the transaction body.
```

Comment:
What we've added is `google.protobuf.Timestamp`, not `uint64`. Which one do we want?

Reply:
changed to timestamp